web/pano_editor.css

```
@@ -2092,3 +2111,502 @@
+  --pano-text-secondary: #a1a1aa;
+  --pano-text-tertiary: #71717a;
+  --pano-space-md: 8px;
+  --pano-space-lg: 12px;
+  --pano-space-xl: 14px;
+  --pano-emphasis-bg: #185fb8;
+  --pano-emphasis-bg-hover: #2672d2;
+  --pano-emphasis-bg-active: #114f9d;
+  --pano-emphasis-text: #fff;
+  --pano-float-bg: rgba(10, 10, 10, 0.82);
+  --pano-float-radius: 8px;
+  box-sizing: border-box;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)